src/LegacyCallHandler.tsx

```diff
@@ -730,7 +730,7 @@ export default class LegacyCallHandler extends EventEmitter {
         }
     };
 
-    private async logCallStats(call: MatrixCall, mappedRoomId: string): Promise<void> {
+    private async logCallStats(call: MatrixCall, mappedRoomId: string | null): Promise<void> {
```

Comment:
I'd have thought it would make more sense to check this in the caller. It's only called at https://github.com/matrix-org/matrix-react-sdk/blob/e2edd21842f275b9fddef3ff912938861de96a0c/src/LegacyCallHandler.tsx#L727; in practice `mappedRoomId` cannot be null there because `matchesCallForThisRoom` will return false if `roomIdForCall(call)` is falsey.